submissions/karelskiy/js-DOM/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link defer rel="stylesheet" href="style.css">
+    <script defer src="script.js"></script>
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="status-bar">
+        <h2>Apple Store</h2>
+    </div>
+    <nav>
+        <ul class="ul-menu">
+            <li><a class="a-menu" href="#">iPhone 11 Pro</a></li>
+            <li><a class="a-menu" href="#">iPhone 11</a></li>
+            <li><a class="a-menu" href="#">iPhone Xr</a></li>
+            <li><a class="a-menu" href="#">iPhone 8</a></li>
+        </ul>
+    </nav>
+    <div class="container">
+        <blockquote>
+            <p>“You can’t connect the dots looking forward; you can only connect them looking backward. So you have to trust that the dots will somehow connect in your future.”</p>
+            <footer>— <cite>Steve Jobs</cite></footer>
+        </blockquote>
+    </div>
+</body>
+
+</html>
```

Comment:
Add an empty line at the end of each file. [Read more](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1) about it